pod-operation/src/main.rs

```diff
@@ -7,31 +7,39 @@ mod components;
 mod demo;
 mod state_machine;
 
+use crate::components::gyro::Mpu6050Sensor;
 use crate::components::lim_temperature::LimTemperature;
 use crate::components::pressure_transducer::PressureTransducer;
 use crate::components::signal_light::SignalLight;
 use crate::state_machine::StateMachine;
-
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
 	tracing::subscriber::set_global_default(FmtSubscriber::default())?;
 
 	let (layer, io) = SocketIo::new_layer();
 
-	let signal_light = SignalLight::new();
-	tokio::spawn(demo::blink(signal_light));
+	// let signal_light = SignalLight::new();
+	// tokio::spawn(demo::blink(signal_light));
 
-	let pressure_transducer = PressureTransducer::new(0x40);
-	tokio::spawn(demo::read_pressure_transducer(pressure_transducer));
+	// let pressure_transducer = PressureTransducer::new(0x40);
+	// tokio::spawn(demo::read_pressure_transducer(pressure_transducer));
 
-	let ads1015 = LimTemperature::new(ads1x1x::SlaveAddr::Default);
-	tokio::spawn(demo::read_ads1015(ads1015));
+	// let ads1015 = LimTemperature::new(ads1x1x::SlaveAddr::Default);
+	// tokio::spawn(demo::read_ads1015(ads1015));
 
-	tokio::spawn(async {
-		let mut state_machine = StateMachine::new(io);
-		state_machine.run().await;
-	});
+	// tokio::spawn(async {
+	// 	let mut state_machine = StateMachine::new(io);
+	// 	state_machine.run().await;
+	// });
 
+	let mut mpu6050_sensor = Mpu6050Sensor::new();
+	tokio::spawn(async move {
+		loop {
+			let (pitch, roll) = mpu6050_sensor.read_accel_gyro();
+			tokio::time::sleep(std::time::Duration::from_millis(100)).await;
+			println!("[{:?} {:?}]", pitch, roll);
+		}
+	});
```

Comment:
Chore: we've been placing our sensor measurement demonstration scripts in `demo.rs`. Could we move this over there to avoid cluttering `main`?